### PR TITLE
closes #3797 fixed toolkit

### LINKED ISSUE #3797: Add tooltip on hover effect on the badges in Individual member profile page.
### Current Behavior

Currently, there is no tooltip on hovering over badges in the individual pages of members on the page https://layer5.io/community/members.
image

### Desired Situation

Add the tooltip while hovering over the badges.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ranjanmangla1**
**Description**

This PR fixes #3797

**Notes for Reviewers**
I have fixed toolkit issues that were coming in @DarrenDsouza7273 's pr #4436

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by DarrenDsouza7273** [comment]
will get back to you once the review is available

**Comment by l5io** [comment]
🚀 Preview for commit 068b5b4e54e65ad9f36c7c1ee4d1909d007e0ec3 at: https://64b1548a5c42291dda6879b8--layer5.netlify.app

**Comment by DarrenDsouza7273** [comment]
Looks better but still would want @Chadha93 @leecalcote to have a look at it

**Comment by Ghat0tkach** [comment]
@ranjanmangla1  , actually the service mesh performance are missing in profiles. Please have a look again , also there is some tiny minute misalignment of arrow for meshery logo

**Comment by Ghat0tkach** [comment]
@ranjanmangla1  , actually the service mesh performance are missing in profiles. Please have a look again , also there is some tiny minute misalignment of arrow for meshery logo

File: src/sections/Community/Member-single/index.js [hunk 1]
```diff
@@ -75,158 +75,43 @@ const MemberSingle = ({ frontmatter }) => {
...
-                        <li>
-                          <Link to="/projects/service-mesh-performance">
-                            <img
-                              className="profile-social-links"
-                              src={smpLogo}
-                              alt="Service Mesh Peformance logo"
-                            />
-                          </Link>
-                        </li>
+                      <Badge name="Meshery" to="/cloud-native-management/meshery" className="profile-social-links" src={mesheryLogo} alt="Meshery logo" firstChild={true}/>
```
Comment (Ghat0tkach) [inline src/sections/Community/Member-single/index.js:78]: add smp badge as well , below this one😅

**Review by Ghat0tkach — CHANGES_REQUESTED** [review]
Add smp badge as well

**Comment by vishalvivekm** [comment]
@ranjanmangla1 Let's discuss it on the websites call. Add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

**Comment by DarrenDsouza7273** [comment]
@ranjanmangla1 hi ranjan  can u please update on this fix

**Comment by Chadha93** [comment]
@ranjanmangla1 Given no response from you here, I'm closing this PR, let's open this for another contributor.

**Pull request closed without merging**